lib/shopify_cli/context.rb

```diff
@@ -357,6 +357,15 @@ def puts(*args)
       Kernel.puts(CLI::UI.fmt(*args))
     end
 
+    # a wrapper around $stderr.puts to allow for easy formatting
+    #
+    # #### Parameters
+    # * `text` - a string message to output
+    #
+    def error(*args)
+      $stderr.puts(CLI::UI.fmt(*args))
```

Comment:
Note that [`.fmt`](https://github.com/Shopify/cli-ui/blob/4766e789e8170ad1b4f49ac6e8121f43e383a1c4/lib/cli/ui.rb#L113) takes the arguments `input, enable_color: enable_color?` and passes `input` down to the [Formatter's initialize](https://github.com/Shopify/cli-ui/blob/4766e789e8170ad1b4f49ac6e8121f43e383a1c4/lib/cli/ui/formatter.rb#L70) which expects the argument to be a string.
If we then call `context.error` with `["a", "b"]` the execution will fail. I'd change the signature to `def error(text)`

Reply:
Good catch! Fixed :)